### DISCUSSION TITLE
mount path must be absolute problem

### DISCUSSION THREAD

**Discussion by chelan-athlanius** [photoprism/photoprism#2491] [Q&A] [answered]
I'm trying to set up a photo prism on a docker container on windows 11. 

I'm at the point of being able to log into the local host but I can't add any photos but now I don't know how to add any of my photos to it. I tried to follow the instructions in the .yml file and I added this file path ( - "~/OneDrive/Desktop/Media/pumpkines" ).

But it just returned this error when I tried to start it.

Error response from daemon: invalid mount config for type "volume": invalid mount path: '~/OneDrive/Desktop/Media/pumpkines' mount path must be absolute
 
What is wrong and what is the proper file path?

**Comment by graciousgrey** [comment, marked as answer]
In case your original files are located in ~/OneDrive/Desktop/Media/pumpkines you need to replace  this line 

 - "~/Pictures:/photoprism/originals" 

from [this](https://dl.photoprism.app/docker/windows/docker-compose.yml) docker-compose example  

with:

- "~/OneDrive/Desktop/Media/pumpkines:/photoprism/originals"

**Comment by vrj93** [comment]
Try using Powershell. This issue arose for me in Git Bash only.

**Reply by Bisyk** [reply]
cmd works as well

**Reply by DaniilRen** [reply]
Yeah, it worked for me
